Foundation/NSStream.swift

```diff
@@ -67,15 +67,41 @@ extension Stream {
     }
 }
 
+extension Stream {
+    internal func _handleStreamEvent(_ event:CFStreamEventType){
+        switch event {
+        case CFStreamEventType.errorOccurred:
+            delegate?.stream(self, handleEvent: Stream.Event.errorOccurred)
+            break
```

Comment:
no particular reason just a  habit from ObjC even though i know that swift switch don't fall through kinda like the referring to self  outside of closure habit  , thanks pointing them out removing them.